src/requestPool/requestPoolManager.js

```diff
@@ -60,7 +112,7 @@ function addRequest (element, imageId, type, preventCache, doneCallback, failCal
 function clearRequestStack (type) {
   // Console.log('clearRequestStack');
```

Comment:
Can you remove this comment while you're updating this file?

Reply:
Done :)